### DISCUSSION TITLE
Officiel Docker image?

### DISCUSSION THREAD

**Discussion by romu70** [go-shiori/shiori#569] [Q&A] [answered]
Hi,
Simple question: is there any official Docker image for Shiori? One can find plenty of images on the Docker Hub, but which one is the official one...if any?

Thanks.

**Comment by fmartingr** [comment, marked as answer]
Yes, you can find the official images in the Github Image Repository: https://github.com/go-shiori/shiori/pkgs/container/shiori

**Comment by romu70** [comment]
Wonderful, thanks a lot!
